lua/entities/gmod_wire_expression2/core/http_cl.lua

```diff
@@ -0,0 +1,157 @@
+--[[
+
+	Clientside HTTP library
+	 - swadical 🍌
+
+]]
+
+--	Clientside HTTP is separate from serverside HTTP:
+--		Serverside HTTP allows greater speeds than clientside HTTP and it is
+--		up to the server owner to decide which extension should be enabled
+--		through a convar (found in core/http.lua)
+
+--	This library will mirror the functionality of core/http.lua as best as
+--	possible, making sure any existing e2s which make use of the http library
+--	are not broken by this extension
+
+-- should we use clientside HTTP requests?
+local cvar_useClient = CreateConVar("wire_expression2_http_client","1",FCVAR_ARCHIVE)
+
+E2Lib.clHTTP = {}
+local lib = E2Lib.clHTTP
+
+lib.netMsgID = "wire_expression2_cl_http" -- open to change
+lib.currentUID = 0
+
+-- number function(void)
+function lib:newUID()
```

Comment:
I was reasonably sure that these UIDs were used to identify the HTTP call when the answer was received from the client, so that the data could be returned to the correct E2(s). That's why I suggested this change to incrementing numbers. Plus if you were to use `WireLib.GetUUID` you'd have to send that whole (much larger) string along with every chunk, which would be meh